env0/resource_environment.go

```diff
@@ -229,6 +229,10 @@ func setEnvironmentConfigurationSchema(d *schema.ResourceData, configurationVari
 	var variables []interface{}
 
 	for _, configurationVariable := range configurationVariables {
+		if configurationVariable.Scope != client.ScopeEnvironment {
```

Comment:
Decided to keep the API as-is.
IMHO it makes more sense for each "consumer" of the API to filter/add whatever it needs.

Reply:
no longer correct. Closing this conversation.